src/symbolic/SymbolicPermutation.jl

```diff
@@ -8,82 +8,92 @@ abstract type PermutationProbabilityEstimator <: SymbolicProbabilityEstimator en
 
 """
     SymbolicPermutation(; τ = 1, m = 3, lt = Entropies.isless_rand) <: PermutationProbabilityEstimator
+    SymbolicWeightedPermutation(; τ = 1, m = 3, lt = Entropies.isless_rand) <: PermutationProbabilityEstimator
+    SymbolicAmplitudeAwarePermutation(; τ = 1, m = 3, A = 0.5, lt = Entropies.isless_rand) <: PermutationProbabilityEstimator
```

Comment:
I would change `PermutationProbabilityEstimator` into `ProbabilityEstimator` in the docstring. The user only needs to know that this is a probability estimator; internal hierarchies are (at the moment) not necessary to expose.